api/v1alpha1/validation/envoyproxy_validate.go

```diff
@@ -294,19 +294,22 @@ func validateFilterOrder(filterOrder []egv1a1.FilterPosition) error {
 	return nil
 }
 
-func validateClusterStatName(clusterStatName string) []error {
-	supportedOperators := map[string]bool{
-		egv1a1.StatFormatterRouteName:       true,
-		egv1a1.StatFormatterRouteNamespace:  true,
-		egv1a1.StatFormatterRouteKind:       true,
-		egv1a1.StatFormatterRouteRuleName:   true,
-		egv1a1.StatFormatterRouteRuleNumber: true,
-		egv1a1.StatFormatterBackendRefs:     true,
-	}
+var supportedOperators = map[string]bool{
+	egv1a1.StatFormatterRouteName:       true,
+	egv1a1.StatFormatterRouteNamespace:  true,
+	egv1a1.StatFormatterRouteKind:       true,
+	egv1a1.StatFormatterRouteRuleName:   true,
+	egv1a1.StatFormatterRouteRuleNumber: true,
+	egv1a1.StatFormatterBackendRefs:     true,
+}
 
+func validateClusterStatName(clusterStatName string) []error {
 	var errs []error
 	re := regexp.MustCompile("%[^%]*%")
 	matches := re.FindAllString(clusterStatName, -1)
+	if len(matches) == 0 {
```

Comment:
my worry is, if this code is reused at route level (in BTP), users may use absolute strings without formatters, should this then be Contains `%` && `len(matches) == 0 ?